src/backend/web/routes/query.js

```diff
@@ -1,29 +1,14 @@
 const express = require('express');
 
+const { queryValidationRules, validateQuery } = require('../validation');
 const { logger } = require('../../utils/logger');
 const { search } = require('../../utils/indexer');
 
 const router = express.Router();
 
-router.get('/', async (req, res) => {
+router.get('/', queryValidationRules(), validateQuery, async (req, res) => {
```

Comment:
Let's not leak implementation details of the validation here by having two separate middleware functions.  In the `validation.js` file, you should amalgamate these two into a single `validateQuery()` function.  Also, the convention we have been using with middleware functions is to call them here and have them return a middleware function (vs. being a function already).

Reply:
Incorporated in [7dd761b](https://github.com/Seneca-CDOT/telescope/pull/1334/commits/7dd761bc7b9281cd31a572a30424e0c41235cb6b)